dcap/ql/types/src/lib.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+
+#![doc = include_str!("../README.md")]
+#![no_std]
+#![deny(missing_docs, missing_debug_implementations, unsafe_code)]
+
+use mc_sgx_core_types::FfiError;
+use mc_sgx_dcap_ql_sys_types::sgx_ql_path_type_t;
+
+/// Paths (location and filename) to be override the default entries.
+#[non_exhaustive]
+#[derive(Eq, PartialEq, Debug)]
+pub enum Path {
```

Comment:
Recommend calling this `PathKind` to disambiguate from `std::path::Path`

Reply:
Done